bundles/org.openhab.binding.hive/README.md

```diff
@@ -0,0 +1,260 @@
+# Hive Binding
+
+<p>
+<img src="https://www.hivehome.com/assets/hive-primary-logo-d6f32b5c6a0a3ecde57e3c6dafead4e6d8a4fb5230177bd85f3aedccf9d59382.svg" alt="Centrica Hive Limited logo" />
```

Comment:
Do not link to images hosted on other domains. If the image disappears it will leave the documentation with a dead image. Either add the image or remove the image. Same for images below.

Reply:
These are from the official website. I linked because I didn't think committing copyrighted images would be a good idea. I'll just remove them all.